.github/actions/remove-mock-release-and-tag/action.yml

```diff
@@ -0,0 +1,57 @@
+name: Mock release cleanup
```

Comment:
this action can be moved to alfresco build tools which is more generic

Reply:
the need does not seem very generic to me though, especially since this repo is public too, so it's easy to reference it from other repos if needed (not sure a lot of repos would need to use this action?)